[2024-11-05T16:19:10.778+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-05T16:19:11.244+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data_pipeline_v2.clean_and_transfer_csv_files.clean_and_transfer_customer manual__2024-11-05T16:19:04.743499+00:00 [queued]>
[2024-11-05T16:19:11.282+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data_pipeline_v2.clean_and_transfer_csv_files.clean_and_transfer_customer manual__2024-11-05T16:19:04.743499+00:00 [queued]>
[2024-11-05T16:19:11.283+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2024-11-05T16:19:11.317+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): clean_and_transfer_csv_files.clean_and_transfer_customer> on 2024-11-05 16:19:04.743499+00:00
[2024-11-05T16:19:11.391+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_data_pipeline_v2', 'clean_and_transfer_csv_files.clean_and_transfer_customer', 'manual__2024-11-05T16:19:04.743499+00:00', '--job-id', '3574', '--raw', '--subdir', 'DAGS_FOLDER/etl2.py', '--cfg-path', '/tmp/tmpf1p3o9rb']
[2024-11-05T16:19:11.393+0000] {standard_task_runner.py:91} INFO - Job 3574: Subtask clean_and_transfer_csv_files.clean_and_transfer_customer
[2024-11-05T16:19:11.387+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=26804) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-05T16:19:11.394+0000] {standard_task_runner.py:63} INFO - Started process 26826 to run task
[2024-11-05T16:19:11.515+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_data_pipeline_v2.clean_and_transfer_csv_files.clean_and_transfer_customer manual__2024-11-05T16:19:04.743499+00:00 [running]> on host 71321b93f95e
[2024-11-05T16:19:11.724+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data_pipeline_v2' AIRFLOW_CTX_TASK_ID='clean_and_transfer_csv_files.clean_and_transfer_customer' AIRFLOW_CTX_EXECUTION_DATE='2024-11-05T16:19:04.743499+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-05T16:19:04.743499+00:00'
[2024-11-05T16:19:11.725+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-05T16:19:11.733+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-11-05T16:19:11.739+0000] {base.py:84} INFO - Using connection ID 'my-gcp-conn' for task execution.
[2024-11-05T16:19:13.077+0000] {gcs.py:344} INFO - File downloaded to /tmp/customer.csv
[2024-11-05T16:19:13.471+0000] {gcs.py:557} INFO - File /tmp/cleaned_customer.csv uploaded to customer.csv in storage-madt-finalproject bucket
[2024-11-05T16:19:13.472+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-05T16:19:13.472+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-05T16:19:13.479+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl_data_pipeline_v2, task_id=clean_and_transfer_csv_files.clean_and_transfer_customer, execution_date=20241105T161904, start_date=20241105T161911, end_date=20241105T161913
[2024-11-05T16:19:13.541+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-05T16:19:13.562+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-05T16:19:13.569+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
